eng/pipelines/corefx-base.yml

```diff
@@ -55,6 +59,11 @@ jobs:
     - template: ../common/templates/job/job.yml
       parameters:
         variables:
+        - name: _dotnetFeedUrl
+          value: https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json
+        - name: _targetOS
```

Comment:
Why is this declaration needed? If we're going to declare a variable to store a parameter value, then we should use the variable elsewhere within the file.

Reply:
"parameter.targetOS" didn't work  when I used it directly in the "publish test assets..." step. The only way I got it to work is to create this variable.

Reply:
Maybe syntax was incorrect? We do it like that elsewhere -- i.e: https://github.com/dotnet/corefx/blob/master/eng/pipelines/helix.yml#L22